spec/models/user_spec.rb

```diff
@@ -1,5 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:first_name) }
```

Comment:
try to group all tests either validation type either a field name

Reply:
I grouped them by validations